easybuild/easyconfigs/q/Qiskit/Qiskit-0.12.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,121 @@
+    }),
+    ('jsonschema', '2.6.0', {
+        'checksums': ['6ff5f3180870836cae40f06fa10419f557208175f13ad7bc26caa77beb1f6e02'],
+    }),
+    ('qiskit-terra', '0.9.0', {
+        'checksums': ['b666ba7d134181f2752e6b340886c97ba98bba7f91e1125bf833d17eda52a2bc'],
+        'modulename': 'qiskit.qobj',
+    }),
+    ('qiskit-ignis', '0.2.0', {
+        'checksums': ['c6bb3c36ede3d625a9d094fddd186d96fb0c532fb507b3f9dbc3bb361f74c38d'],
+        'modulename': 'qiskit.ignis',
+    }),
+    ('fastdtw', '0.3.2', {
+        'checksums': ['09a24d62a8eb72f87c03692dffae4b8d8499b53f503e9abce5e717c5b705dff9'],
+    }),
+    ('dlx', '1.0.4', {
+        'checksums': ['ef75bc9d590216ebde7d4811f9ae6b2d6c6dc2a54772d94ae13384dc517a5aae'],
+    }),
+    ('docloud', '1.0.375', {
+        'checksums': ['996d55407498fd01e6c6c480f367048f92255e9ca9db0e9ea19aaef91328a441'],
+    }),
+    ('docplex', '2.10.154', {
+        'checksums': ['34a36ff25dd44b72cfe76e01dd0658eb7a451b0c3a52ef9f98e8e38fed83d537'],
+    }),
+    ('qiskit-aqua', '0.6.0', {
+        'checksums': ['85203ffb7d815bfc00606e7792fe9d1d233546a50032c67a72cad1c69008af6d'],
+        'modulename': 'qiskit.aqua',
+    }),
+    ('qiskit-chemistry', '0.5.0', {
+        'checksums': ['cb1dafa0753acfc98cb258ed773d7506d96f16cd203ed2f2d18bb8f10346ede0'],
+        'modulename': 'qiskit.chemistry',
+    }),
+    ('websockets', '7.0', {
+        'checksums': ['08e3c3e0535befa4f0c4443824496c03ecc25062debbcf895874f8a0b4c97c9f'],
+    }),
+    ('ntlm-auth', '1.4.0', {
+        'checksums': ['350f2389c8ee5517f47db55a36ac2f8efc9742a60a678d6e2caa92385bdcaa9a'],
+    }),
+    ('requests_ntlm', '1.1.0', {
+        'checksums': ['9189c92e8c61ae91402a64b972c4802b2457ce6a799d658256ebf084d5c7eb71'],
+    }),
+    ('nest-asyncio', '1.0.0', {
+        'source_tmpl': 'nest_asyncio-%(version)s.tar.gz',
+        'checksums': ['bd1cb7df2ea979e57d8ad02493ad85f9afbf1fcea3dfe34239da8c0dda98087e'],
+    }),
+    ('qiskit-ibmq-provider', '0.3.2', {
+        'checksums': ['afcc7ea4a1578a61316d1d32eb9a5111e459ddb222f6023c7db9f13887a645cc'],
+        'modulename': 'qiskit.providers.ibmq',
+    }),
+    ('scikit-build', '0.10.0', {
+        'checksums': ['7342017cc82dd6178e3b19377389b8a8d1f8b429d9cdb315cfb1094e34a0f526'],
+        'modulename': 'skbuild',
+    }),
+    ('qiskit-aer', '0.3.0', {
+        'checksums': ['a0d26376199932bd26ef9a682b303a471ed2c9fe2e97cc05f01e373291d18e2b'],
+        'modulename': 'qiskit.providers.aer',
+    }),
+    ('qiskit', version, {
+        'checksums': ['9d133b0fc4fd5be5eb67131aee8c572b73e8dec0607ef5a07977654d565950ef'],
+    }),
+]
+
+postinstallcmds = [
+    # qiskit/providers/aer/VERSION.txt does not get installed, but is required by qiskit/providers/aer/version.py
+    "echo %s > %%(installdir)s/lib/python%%(pyshortver)s/site-packages/qiskit/providers/aer/VERSION.txt" % local_aerver,
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```

Comment:
So, what's the deal with sanity_check_paths and PythonBundle now again? I vaguely recall this being part of the block. Maybe?

Reply:
Yes, indeed, this doesn't need to be here, it's exactly the same as the default. So, removing...